### ISSUE TITLE
Request: prompt after completion

### ISSUE THREAD

**Issue by Efreak** [inorichi/tachiyomi#509]
There is a setting that controls how many chapters should stay downloaded after you read them. This is nice, as it allows me to it keep 1-2 chapters to go back to, however it would be nice if after reading the last chapter and exiting the reader it prompted me to delete all downloaded chapters, change the tags/categories, and/or unfavorite it, possibly depending on whether the metadata states that the work is completed/ongoing.

**Comment by inorichi** [comment]
The number of chapters can only be known with tracking services. I can't just use the manga metadata because a manga can be completed and with only 1 chapter translated/available. Though I'm not totally against this feature relying on the tracking services.

**Comment by Efreak** [comment]
I have it set to keep the previous chapter downloaded. as sometimes I go too far when switching pages rapidly, or I want to go back a chapter to confirm something. However, many mangas update infrequently, so it's not useful to keep the last chapter around when there's nothing new (it wastes my poorly-organized storage space). So when I'm done reading all the *currently available* chapters, I would like to have that last one deleted, as it's only useful to me when there's something to read, so I don't have to manually delete the last chapter every time I finish an updated manga.

The metadata is not always useful anyways--finished/completely available mangas are frequently listed as being ongoing.

TL;DR: when I say *last chapter*, I mean *last __known__ chapter*, or *last chapter in chapter list*.
